SIT.Manager.Updater/Program.cs

```diff
@@ -90,8 +92,29 @@
 Console.WriteLine("\nDownload complete.");
 Console.WriteLine("Creating backup of SIT.Manager");
 
+DirectoryInfo releasePath = new(Path.Combine(tempPath, "Release"));
+releasePath.Create();
+using (ZipArchive archive = ZipArchive.Open(zipPath))
+{
+    foreach (ZipArchiveEntry entry in archive.Entries)
+    {
+        if (entry.IsDirectory)
+        {
+            continue;
+        }
+        else
+        {
+            entry.WriteToDirectory(releasePath.FullName, new ExtractionOptions()
+            {
+                ExtractFullPath = true,
+                Overwrite = true
+            });
+        }
+    }
+}
+
```

Comment:
Now we found what's going wrong I think we leave using the SharpCompress package but this could be condensed down to this:
```csharp
using (ZipArchive archive = ZipArchive.Open(zipPath))
{
    archive.ExtractToDirectory(releasePath.FullName);
}
```

Reply:
true, true. I'll fix that ^^